test/callbacks.jl

```diff
@@ -9,9 +9,8 @@ using Test
 struct DummyCallbackData end
 
 @testset "LazyConstraint" begin
-    mock = MOI.Utilities.MockOptimizer(
-        MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}())
-    )
+    mock =
+        MOI.Utilities.MockOptimizer(MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}()))
```

Comment:
This seems like a bug. The formatting violates the line length, and takes good formatting and makes it bad. There are more examples below.

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/338